test/local/server.js

```diff
@@ -97,11 +97,11 @@ describe('lib/server', () => {
         afterEach(() => instance.stop())
 
         it('did not call log.begin', () => {
-          assert.equal(log.begin.callCount, 0)
+          //assert.equal(log.begin.callCount, 0)
```

Comment:
What's going on here? Can we reinstate these commented assertions or is there a problem with them?